apps/angular/src/app/features/home/anime-table/anime-table.component.html

```diff
@@ -0,0 +1,46 @@
+@if(animePage$ | async; as animePage) {
+<section class="mat-elevation-z8 anime__table-container">
+	<table mat-table [dataSource]="animePage !== null ? animePage.items : []" [trackBy]="trackAnimeById">
+		<ng-container matColumnDef="image">
+			<th mat-header-cell *matHeaderCellDef class="anime__cell">Image</th>
+			<td mat-cell *matCellDef="let anime">
+				<div class="anime__image-container">
+					<img class="anime__image" [src]="anime.image" alt="anime.titleJpn">
```

Comment:
It would be better if `alt` also handle case when `titleJpn` is null. Consider this
```suggestion
					<img class="anime__image" [src]="anime.image" [alt]="anime.titleJpn ?? 'Anime image'">
```